pkg/metrics/metrics.go

```diff
@@ -158,6 +163,11 @@ var clusterStatusCapacityLabels = []string{
 	LabelCloudProvider,
 }
 
+var KasFleetManagerClusterProviderResourceQuotaConsumedLabels = []string{
```

Comment:
Remove KasFleetManager prefix to be consistent with other variables. 
```suggestion
var ClusterProviderResourceQuotaConsumedLabels = []string{
```